content/en/tracing/trace_retention_and_ingestion/_index.md

```diff
@@ -0,0 +1,145 @@
+| Tracers Configuration            | Will show `Default` unless changed by using the instructions in-app to configure the tracer. See [Change the Default Ingestion Rate](#change-the-default-ingestion-rate) for more information. If all hosts with this service deployed are configured to send a specific volume of traces, this indicator will display as `Fully Configured`.  If only a portion of hosts with this service deployed are configured, the label will instead show `Partially Configured`.   |
+| Dropped Spans                |  The percentage of incoming spans dropped by the Datadog Agent.  If this percent is higher than 0%, the service can be configured by clicking anywhere on the service row.  See [Change the Default Ingestion Rate](#change-the-default-ingestion-rate) for more information.     |
+| Traces Ingested per Second                |   Average number of traces per second ingested into Datadog for the service over the selected time period.   |
+| Spans Ingested            | Number of spans ingested by Datadog over the selected time period.        |
+
+### Change the Default Ingestion Rate
+
+{{< img src="tracing/trace_indexing_and_ingestion/ChangeIngestRate3.gif" style="width:100%;" alt="Change the Data Ingestion Rate" >}}
+
+To specify that a specific percentage of a service's traffic should be sent, add a generated code snippet to your tracer configuration for that service.
+
+1. Select the service you want to change the ingested span percent for.
+2. Choose the service language.
+3. Choose the desired ingestion percentage.
+4. Apply the appropriate configuration generated from these choices to the indicated service and redeploy.
+5. Confirm on the Data Ingestion page that your new percentage has been applied.
+
+#### (Recommended) Setting the global ingestion rate to 100%
+
+In order to ingest 100% of your traces in Datadog for all services for live search and analytics as well as to have the most control with retention filters, Datadog recommends configuring all services to send 100% of traces by default.
+
+This can be done with one configuration on every service instrumented with a Datadog tracing library by setting the environment variable below in the tracer configuration:
+
+```
+DD_TRACE_SAMPLE_RATE=1.0
+```
+
+### Traces dropped before ingestion
+
+If you have not set the environment variable configuration `DD_TRACE_SAMPLE_RATE=1.0` for Tracing without Limits and
+- your applications generate above 50 traces per second
+- your applications send intermittent traffic loads
+- your applications traces are large in size or otherwise have complicated trace payloads
+
+This may make them not always send 100% of your traces to Datadog.  In this case, some traces will be dropped en route to Datadog.  These traces will be dropped in the Datadog Agent, *after* stats are computed, so that metrics calculated will be based on 100% of your traces.
```

Comment:
```suggestion
You won't get 100% trace ingestion if you have not set the environment variable configuration `DD_TRACE_SAMPLE_RATE=1.0` for Tracing without Limits, and:
- your applications generate above 50 traces per second;
- your applications send intermittent traffic loads; or
- your applications traces are large in size or otherwise have complicated trace payloads.

In this case, some traces will be dropped by the Datadog Agent *after* stats are computed, so that metrics calculated will be based on 100% of your traces.
```